src/app.js

```
@@ -1 +1,20 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('node:fs');
+
+function copy() {
+  const [sourceFile, destinationFile] = process.argv.slice(2);
+
+  if (process.argv.length !== 4) {
+    console.error('You need provide 4 arg');
+
+    return;
+  }
+
+  fs.copyFile(sourceFile, destinationFile, (error) => {
+    console.error(error);
```

The error handling should throw an error using `console.error()` as specified in the task requirements.